dashboard/test/models/concerns/pd/form_test.rb

```diff
@@ -32,6 +62,16 @@ class Pd::FormTest < ActiveSupport::TestCase
     assert_equal ["can't be blank"], form.errors.messages[:form_data]
   end
 
+  test 'pd form only validates form data when changed' do
+    form = DummyFormWithRequiredFields.new
+    form.form_data = {firstField: "foo"}.to_json
+    form.save
+
+    assert form.valid?
+    form.form_data = {firstField: "bar"}.to_json
+    assert_equal false, form.valid?
+  end
+
```

Comment:
I don't get what this is testing. Why is foo valid and bar invalid? Also I'd like to explicitly see the case where an "invalid" form data is grandfathered in via `save(validate: false)`, then shown to be valid unless modified.

nit: `refute` instead of `assert_equal false,`